tests/phydat_unix/Makefile.ci

```diff
@@ -0,0 +1 @@
+BOARD_INSUFFICIENT_MEMORY :=
```

Comment:
You can create this list by running 

    make create-Makefile.ci

Reply:
Is there documentation for this somewhere? It didn't work for me in the form you suggested (I had to add a `BOARD=`) or using `BUILD_IN_DOCKER=1`. It seemed to work by manually running things in the Docker image, however, so I hope it's correct.